Bluepill-cli/Bluepill-cli/Bluepill/Reporters/BPTreeParser.m

```diff
@@ -380,7 +389,9 @@ - (void)parseLine:(nullable NSString *)line {
                 if (!testCaseLogEntry.passed) {
                     [self onTestCaseFailedWithName:testCaseLogEntry.testCaseName inClass:testCaseLogEntry.testCaseClass
                                             inFile:testCaseLogEntry.filename onLineNumber:testCaseLogEntry.lineNumber
-                                      wasException:testCaseLogEntry.unexpected];
+                                      wasException:testCaseLogEntry.unexpected
+                                    saveScreenshot:!screenshotAlreadySaved];
+                    screenshotAlreadySaved = YES;
```

Comment:
hmm, is this flag actually used for dedup screenshots? The regexes are supposed to match one type of failure.

Reply:
yea, if so I will remove it ..